bisect_ppx-ocamlbuild.opam

```diff
@@ -0,0 +1,24 @@
+name: "bisect_ppx-ocamlbuild"
```

Comment:
@rgrinberg should `version` also be avoided? I guess it's always known when installing from OPAM, and when pinning it's inferred as `dev` anyway AFAIK.